modules/argocd-installing-argocd.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-installing-argocd_{context}"]
+= Installing ArgoCD
+
+To use ArgoCD to manage your cluster and application configuration files, you must install it after you install your {product-title} cluster.
+
+[NOTE]
+====
+These steps will be deprecated when the ArgoCD Operator is added to OperatorHub.
+====
+
+.Prerequisites
+
+* You installed a {product-title} cluster and the `oc` command line tool.
+* You installed the `oc` command line tool.
+* You have the link:https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/[kubeconfig] file for your cluster. The default `kubeconfig` file is in the `~/.kube/config` folder on the machine that you installed your cluster from.
+
+.Procedure
+
+. Create a new project for ArgoCD:
++
+----
+$ oc new-project argocd
+----
+
+. Apply the ArgoCD manifest file to your cluster:
++
+----
+$ oc apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
+----
+
+. Create a route for the ArgoCD service:
++
+----
+$ oc create route passthrough --service=argocd-server
+----
+
+////
+# but this does not seem to work for console logins...
+#oc apply -n argocd -f argocd.yaml
+#oc create route edge --service=argocd-server
+////
+
+
+. Get the ArgoCD `admin` password:
++
+----
+$ ARGO_ADMIN_PASS=`kubectl get pods -n argocd -l app.kubernetes.io/name=argocd-server -o name | cut -d'/' -f 2`
+----
+
+. Log in to your ArgoCD instance:
++
+----
+$ ARGO_ROUTE=`oc get route argocd-server -n argocd -o jsonpath='{.spec.host}'`
+$ argocd login $ARGO_ROUTE:443 --username admin --password $ARGO_ADMIN_PASS --insecure
+----
+
+. Change the ArgoCD password for the `admin` user:
++
+----
+$ argocd account update-password
+----
++
+[NOTE]
+====
+By default, ArgoCD has only an `admin` user.
+====
+
+. Optional: Configure SSO so that more users can manage ArgoCD. See link:https://argoproj.github.io/argo-cd/operator-manual/sso/[SSO] in the ArgoCD documentation.
```

Comment:
You might want to add this: https://blog.openshift.com/openshift-authentication-integration-with-argocd/